### PR TITLE
Ai

### PR DESCRIPTION
**Pull request by contfire**
add gpu docker

### CODE REVIEW

**Comment by XinShuYang** [comment]
@cntnfire Please update title and commit message.

File: ai/gpu docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM nvidia/cuda:12.2.0-devel-ubuntu20.04
```
Comment (XinShuYang) [inline ai/gpu docker/Dockerfile]: Is this dockerfile only for training? We should use different dockerfile for training and prediction.

File: ai/gpu docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=CST-8
+
+RUN apt-get update -y && apt-get install -y python3.9 python3.9-distutils curl \
+  && apt-get install git -y \
+  && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
+  && python3.9 get-pip.py \
+  && pip3 install fschat[model_worker,webui] fschat torch torchvision torchaudio \
+  && pip3 install accelerate  transformers[sentencepiece] transformers sentencepiece flash_attn
```
Comment (XinShuYang) [inline ai/gpu docker/Dockerfile]: Please add readme for dockerfile as well.

File: ai/github_issue_handling/issue_clawer.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
```
Comment (XinShuYang) [inline ai/github_issue_handling/issue_clawer.py:2]: This PR is only for gpu dockefile, we can move filter script change to https://github.com/XinShuYang/antrea/pull/5/files @cntnfire @Createsnow

Comment (Createsnow) [reply ai/github_issue_handling/issue_clawer.py:2]: It seems that my commit history has also been re-submitted. Perhaps @cntnfire should commit only the specified modified files?

Comment (XinShuYang) [reply ai/github_issue_handling/issue_clawer.py:2]: @Createsnow  For all filter related commits, you should post them in this PR #5 as they are all about data processing scripts. This PR #10 is only for GPU dockerfile.

File: ai/gpu docker/Readme.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+The Dockerfile works for both GPU training and prediction. Please build it with the docker version of Nvidia-docker2.
```
Comment (XinShuYang) [inline ai/gpu docker/Readme.txt]: Please add detailed commands to build image and run container for both training and prediction.

**Comment by contfire** [comment]
add the ai server deploy

**Pull request closed without merging**